src/examples/keesom.cpp

```diff
@@ -15,7 +16,7 @@ int main() {
   auto b=a;                    // duplicate particle...
   b.x() = 0.9_angstrom;        // ...and separate by 0.9 angstrom
 
-  Potential::DipoleDipole u(298.15_K, 80.); // pair potential
```

Comment:
For a simple example it may be instructive to have a constructor that takes basic input like temp. and diel. In this case, it also shows how to use units.